lib/config/config.js

```diff
@@ -439,10 +454,82 @@ function loadConfigFile (configFile) {
   return null
 }
 
+function printLoggingPreamble (config, sources, configFilePath) {
```

Comment:
Perhaps link to https://github.com/elastic/apm/blob/main/specs/agents/logging.md#logging-preamble in a comment here.